content/en/docs/guides/complex-component-structure-deployment.md

```diff
@@ -0,0 +1,1069 @@
+    ├── sha256.893be914eebd8230ef848ea82b3433c6201152f5d9925e7b5b8d68e0cec7133e
+    ├── sha256.92991cf391167c928f3afe6891001f3dd325b64ce800cf34fad4c038141fc57f
+    ├── sha256.98ca4d46130f5c09a704b3d8ee9af94de3c0ac73d7e990df53e64606c418fea8
+    ├── sha256.a779270c2fea310835d3125de90e089e423c9730a98f1acdda328470d21fced0
+    ├── sha256.a7dd532f80e8417ed33cf0c97328582847017895fc5146e499bdf4c94a9d17b5
+    ├── sha256.cae4365f264251c616210707aa4765bd95f23fd22f98abc68bae9f58d6e4506d
+    ├── sha256.ee79c92bbcce9e7a98f07c6577fd56dd45cf6f7c2d3115216ee249f42119030e
+    └── sha256.f6a82a23220752c232e5f66ce46f0be28b27a5af19474072c77dac6d1feb0c16
+
+2 directories, 19 files
+```
+
+These blobs are OCI layers. They contain information about our podinfo application. If we cat a random blob we get
+something like this:
+```
+cat sha256.3c9c902ce013ca070a29634e4603c90063c96df632ef2c8e6b4447aaeb70b67e
+{"componentDescriptorLayer":{"mediaType":"application/vnd.ocm.software.component-descriptor.v2+yaml+tar","digest":"sha256:699ea8628e39256048cd1687c496fe64999a41f16f200ef5ce938ee9f19c37f0","size":2560}}%
+```
+
+Next, we transfer this component to a location of your choice. For example:
+
+```
+ocm transfer component ./transport-archive ghcr.io/skarlso/mpas-demo-component
+transferring version "ocm.software/podinfo:1.0.2"...
+...adding component version...
+transferring version "ocm.software/podinfo/backend:1.0.0"...
+...resource 0...
+...resource 2...
+...adding component version...
+transferring version "ocm.software/podinfo/frontend:1.0.0"...
+...resource 0...
+...resource 2...
+...adding component version...
+transferring version "ocm.software/redis:1.0.0"...
+...resource 0...
+...resource 2...
+...adding component version...
+4 versions transferred
+```
+
+Now, we should have a component that we can use and deploy.
+
+### ComponentVersion
+
+We start by creating a pull secret since the component that we just transferred is private. This pull secret will be
+used by OCM to access this package in ghcr. To create the secret, run:
+
+```
+kubectl create secret docker-registry pull-secret -n ocm-system \
+    --docker-server=ghcr.io \
+    --docker-username=$GITHUB_USER \
+    --docker-password=$GITHUB_TOKEN \
+    --docker-email=$GITHUB_EMAIL
+```
+
+We start by applying the components version main object to reconcile the component.
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: ComponentVersion
+metadata:
+  name: podinfocomponent-version
+  namespace: ocm-system
+spec:
+  component: ocm.software/podinfo
+  interval: 10m0s
+  references:
+    expand: true
+  repository:
+    url: ghcr.io/skarlso/mpas-demo-component # this should be where you transferred the component.
+    secretRef:
+      name: pull-secret
+  version:
+    semver: 1.0.2
+```
+
+This will reconcile the version into the cluster for the rest of objects to gather data from. If everything was successful
```

Comment:
```suggestion
This will reconcile the `ComponentDescriptor` for the specific version, making the component metadata available for other Kubernetes resources to consume. If everything was successful
```